strings.txt

```diff
@@ -14412,3 +14412,410 @@
+    cs = Počátek cesty
+    it = Il punto di partenza
+    nb = Avreisested
+    zh-Hant = 出發地點
+    zh-Hans = 出发地点
+    th = จุดที่ออกเดินทาง
+    ja = 出発地
+    ro = Punctul de plecare
+    ar = نقطة المغادرة
+    sk = Východiskový bod
+    es = El punto de origen
+    pl = Miejsce wyjazdu
+    nl = Vertrekpunt
+    pt = O ponto de partida
+
+  [p2p_to]
+    en = To
+    ru = Пункт назначения
+    fr = Le lieu de destination
+    da = Bestemmelsesstedet
+    id = Titik tujuan
+    ko = 도착 지점
+    sv = Målplats
+    tr = Varış noktası
+    uk = Пункт призначення
+    vi = Điểm đích
+    hu = Az érkezés helye
+    de = Ankunftspunkt
+    fi = Päämäärä
+    cs = Místo určení
+    it = Il punto di destinazione
+    nb = Destinasjonssted
+    zh-Hant = 目的地點
+    zh-Hans = 目的地点
+    th = จุดที่เป็นปลายทาง
+    ja = 目的地
+    ro = Punctul de sosire
+    ar = نقطة الوصول
+    sk = Cieľový bod
+    es = El punto de destino
+    pl = Miejsce docelowe
+    nl = Bestemmingspunt
+    pt = O ponto de destino
+
+  [p2p_start]
+    en = Start
+    ru = Начать движение
+    fr = Démarrer
+    da = Start
+    id = Mulai
+    ko = 시작
+    sv = Start
+    tr = Başla
+    uk = Почати рух
+    vi = Bắt đầu
+    hu = Indítás
+    de = Start
+    fi = Aloita
+    cs = Start
+    it = Inizia
+    nb = Start
+    zh-Hant = 開始
+    zh-Hans = 开始
+    th = เริ่มต้น
+    ja = 開始
+    ro = Start
+    ar = البداية
+    sk = Štart
+    es = Empezar
+    pl = Start
+    nl = Beginnen
+    pt = Iniciar
+
+  [p2p_planning]
+    en = Planning…
+    ru = Планируем…
+    fr = En cours de planification…
+    da = Planlægger…
+    id = Merencanakan…
+    ko = 계획 중…
+    sv = Planerar…
+    tr = Planlanıyor…
+    uk = Плануємо…
+    vi = Đang hoạch định…
+    hu = Tervezés…
+    de = Route wird geplant…
+    fi = Lasketaan…
+    cs = Plánování…
+    it = Pianificazione…
+    nb = Planlegger…
+    zh-Hant = 計劃中…
+    zh-Hans = 计划中…
+    th = กำลังวางแผน…
+    ja = 作成中…
+    ro = Se planifică…
+    ar = جار التخطيط…
+    sk = Plánovanie…
+    es = Planeando…
+    pl = Planowanie…
+    nl = Aan het plannen…
+    pt = Planejando…
+
+  [p2p_from_here]
+    en = From
+    ru = Отсюда 
+    fr = Depuis
+    da = Fra
+    id = Dari
+    ko = 출발지
+    sv = Från
+    tr = Başlangıç
+    uk = Звідси
+    vi = Từ
+    hu = Innen
+    de = Von
+    fi = Lähtöpaikka
+    cs = Z
+    it = Da
+    nb = Fra
+    zh-Hant = 從
+    zh-Hans = 从
+    th = จาก
+    ja = 出発地
+    ro = Din
+    ar = من
+    sk = Z
+    es = Desde
+    pl = Z
+    nl = Van
+    pt = De
+
+  [p2p_to_here]
+    en = To
+    ru = Сюда
+    fr = Jusqu'à
+    da = Til
+    id = Ke
+    ko = 도착지
+    sv = Till
+    tr = Hedef
+    uk = Сюди
+    vi = Đến
+    hu = Ide
+    de = Nach
+    fi = Määränpää
+    cs = Do
+    it = A
+    nb = Til
+    zh-Hant = 到
+    zh-Hans = 到
+    th = ถึง
+    ja = 目的地
+    ro = În
+    ar = إلى
+    sk = Do
+    es = A
+    pl = Do
+    nl = Naar
+    pt = Para
+
+  [p2p_only_from_current]
```

Comment:
To Alex: you can start the follow mode only if you navigate from the current location, otherwise there is no point in navigating if you are miles away from the actual route.